packages/react/src/SelectPanel/SelectPanel.tsx

```diff
@@ -93,6 +93,10 @@ interface SelectPanelBaseProps {
     variant: 'empty' | 'error' | 'warning'
   }
   onCancel?: () => void
+  orderSelectedFirst?: boolean
```

Comment:
```suggestion
  showSelectedOptionsFirst?: boolean
```

Just a naming idea, totally get if it doesn't work as well as `orderSelectedFirst` 👍

Reply:
done!